plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -0,0 +1,196 @@
+package pgbouncer
+
+import (
+	"bytes"
+	"database/sql"
+	"regexp"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"github.com/lib/pq"
```

Comment:
I believe so. I think it is worth the effort to investigate. if I'm right then implementing it will also resolve an outstand feature request for the postgresql_extensible plugin.